ssc/cmod_pvsamv1.cpp

```diff
@@ -946,6 +972,8 @@ static var_info _cm_vtab_pvsamv1[] = {
         // annual_ac_gross
         { SSC_OUTPUT, SSC_NUMBER, "annual_ac_inv_eff_loss_percent", "AC inverter efficiency loss", "%", "", "Loss", "", "", "" },
         { SSC_OUTPUT, SSC_NUMBER, "annual_ac_wiring_loss_percent", "AC wiring loss", "%", "", "Loss", "", "", "" },
+        { SSC_OUTPUT, SSC_NUMBER, "annual_subhourly_clipping_loss_percent", "Subhourly clipping correction loss percent", "%", "", "Loss", "", "", "" },
```

Comment:
@cpaulgilman curious on your thoughts if this output should be reported if the subhourly clipping loss model isn't enabled. I don't have a strong opinion one way or the other for the annual results.